docs/FAQ.md

```diff
@@ -191,25 +178,19 @@ $ brew link foo
 Linking /usr/local/Cellar/foo/0.1… 17 symlinks created
 ```
 
-## Why was a formula deleted?
-Use `brew log <formula>` to find out! Likely because it had unresolved issues or
-our analytics identified it was not widely used.
+## Why was a formula deleted or disabled?
+Use `brew log <formula>` to find out! Likely because it had [unresolved issues](Acceptable-Formulae.md) or [our analytics](Analytics.md) identified it was not widely used.
```

Comment:
Is lack of widespread use an independent reason formulae are deleted/disabled? Coupled with other issues it may make sense to delete it, but I didn't think low usage alone was enough to warrant formula removal.